crates/protocol/src/operator/state.rs

```diff
@@ -334,6 +372,25 @@ impl LogState {
         Ok(())
     }
 
+    fn validate_namespace(
+        &mut self,
+        name: &str,
+        definition: NamespaceDefinition,
+    ) -> Result<(), ValidationError> {
+        let name = name.to_lowercase();
+
+        if !PackageId::is_valid_namespace(&name) {
```

Comment:
Check that name is lowercase in this function

Reply:
Waiting for the outcome of the [discussion on package names and namespaces](https://github.com/bytecodealliance/registry/discussions/231)

Reply:
Modified to keep the case for package names and namespaces for presentation and code generation. But enforces uniqueness in a case insensitive way.